assets/hip-1056/protobuf/stream/output/smart_contract_service.proto

```diff
@@ -129,33 +103,119 @@ message ContractSystemDeleteOutput {}
  * the original transaction.
  */
 message EthereumOutput {
+     /**
+      * A receipt for an Ethereum Transaction execution
+      */
+     EVMTransactionReceipt evm_transaction_receipt = 1;
+}
+
+/**
+ * EVM transaction execution log storage details
+ * Details maps to the log object in eth_getTransactionReceipt response without repeating info already available in the input transaction
+ */
+message EVMTransactionLog {
+     /**
+      * The address of the contract emitting the log
+      */
+     string address = 1;
+
+     /**
+      * The sequence index of the log event in this contract execution
+      */
+     int32 log_index = 2;
```

Comment:
Need to correct the comment to be in block.
Also need to confirm if CN also has this available and if this matches the order it's written in the block stream. If so then this can be removed.
If the order of write is not matching the index in the block then this should be maintained

Reply:
right now in mirrornode log_index is the index of a log in its EVM transaction. Is this something we try to fix?

Reply:
Hmm, the EVM defined it as across block so i think so.
Ses https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getfilterchanges

Some examples
- https://etherscan.io/tx/0xcf833ccd3f7ef7b4e39966b72a3d1a1fcdd6c0e339241ca03102ac3bacf47468#eventlog
- https://etherscan.io/tx/0x295425b3d5a1c7344a8fdde9e8004035897509d5bae953a88d588f0b7049e547#eventlog
- https://etherscan.io/tx/0x6a5dc82b3df0db293459bf13a6689d7709b75648a0662a99000c83974889ec69#eventlog
- https://etherscan.io/tx/0x19757f097b0c89024441184ed29b2246fb86fd7528ec9de7c38e2a626d1c9ee1#eventlog

Reply:
dang, alright, so we need to fix it

https://github.com/hiero-ledger/hiero-mirror-node/blob/bc7cd8d3b0f261001cdd9f4c2735cd1381c78b07/hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/ContractResultServiceImpl.java#L279-L290